### PR TITLE
Adding week 5 blog post

### PR DESCRIPTION
**Pull request by xtanion**
The blog is visible in `blos.html`
image

### CODE REVIEW

File: docs/source/posts/2022/2022-07-19-week5-shivam.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- Working on importing different primitive modes, the current glTF importer can render only triangles.
+
+
+What is coming up next week?
+----------------------------
+
+This week I'll be working on the following:
+
+- Get the exporting PR merged by this week. 
+- Loading the animations (simple, morph, and skeletal) data as dictionaries from the glTF model so that It can be sent to the timeline.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-07-19-week5-shivam.rst:24]: `it` instead

**Review by ganimtron-10 — COMMENTED** [review]
Hello @xtanion ,
Short and Nice Blog!! LGTM.
Thanks!

**Comment by m-agour** [comment]
Hello @xtanion,
Great job. I fount no typos but have one suggestion for improving readability: can you put a link to the PRs you are referencing in your blog post to allow the reader to have access to it?

**Comment by xtanion** [comment]
Thanks for the review @ganimtron-10 and @m-agour

**Comment by skoudoro** [comment]
Thanks all, merging!

**Pull request merged**